From 4d8aa900239bb666898050f295f23c35d0e0e093 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 18 Jan 2017 21:50:03 +0000
Subject: [PATCH] Support a stateless configuration

Solus uses stateless directories, as such Flatpak should not be using
the sysconfdir for default configuration data.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 Makefile.am                   | 2 +-
 system-helper/Makefile.am.inc | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 24b2528..3e5a221 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -119,7 +119,7 @@ completiondir = $(datadir)/bash-completion/completions
 completion_DATA = completion/flatpak
 EXTRA_DIST += $(completion_DATA)
 
-profiledir = $(sysconfdir)/profile.d
+profiledir = $(datadir)/defaults/etc/profile.d
 profile_DATA = flatpak.sh
 EXTRA_DIST += \
 	profile/flatpak.sh.in \
diff --git a/system-helper/Makefile.am.inc b/system-helper/Makefile.am.inc
index e8a04f3..2787c92 100644
--- a/system-helper/Makefile.am.inc
+++ b/system-helper/Makefile.am.inc
@@ -8,7 +8,7 @@ dbussystemservicedir = $(datadir)/dbus-1/system-services
 service_in_files += system-helper/org.freedesktop.Flatpak.SystemHelper.service.in
 dbussystemservice_DATA = system-helper/org.freedesktop.Flatpak.SystemHelper.service
 
-dbusconfdir = ${sysconfdir}/dbus-1/system.d
+dbusconfdir = ${datadir}/dbus-1/system.d
 dbusconf_DATA = system-helper/org.freedesktop.Flatpak.SystemHelper.conf
 
 service_in_files += system-helper/flatpak-system-helper.service.in
-- 
2.11.0

